Documentation/ceph-upgrade.md

```diff
@@ -259,10 +259,31 @@ At this point, your Rook operator should be running version `rook/ceph:v1.0.0`
 
 Verify the Ceph cluster's health using the [health verification section](#health-verification).
 
+### 6. Update the Mon Ports
+
+The port on which the mons are listening has changed in the 1.0 release. Previously, the mons were listening on
+port 6790 by default. In 1.0 now they are expected to be listening on port 6789. To ensure an uninterrupted
+upgrade to 1.0, the operator configures the existing mons to continue listening on port 6790.
+
+**Before upgrading to Ceph Nautilus,** it is recommended that the mons be re-configured to listen on
+the new deafult port 6789. Instead of changing the port, the recommended process is to failover the mons,
+which will create new mons on port 6789. While the operator will automate most of this process, there are
+several steps required to induce the operator to failover a mon.
+1. Cause the mon to fail by setting the replicas on the mon deployment to zero. For example, if your mon is named `mon-a`:
+   - `kubectl -n rook-ceph edit deploy rook-ceph-mon-a`
+1. Find the line with `replicas: 1` and change it to `replicas: 0`. Save the change and exit the editor.
+1. Wait for 5-10 minutes for the operator to fail over the mon. You will see messages in the operator log that the mon is down and will be failed over after a timeout.
+1. After the timeout, a new mon will be started and the old mon deployment will be automatically removed.
+1. Confirm in the toolbox that all mons are in quorum: `ceph status`
+1. Repeat steps 1-5 for each of the old mons
+
+Note that clients connected with the Rook flex driver will be automatically updated when the mons are failed over.
+No intervention is needed for the clients to find the new mons.
+
 
 ## Ceph Version Upgrades
 Rook 1.0 is the last Rook release which will support Ceph's Luminous (v12.x.x) version. Users are
-advised to upgrade to Mimic (v13.x.x) or Nautilus (v14.x.x) now.
+advised to upgrade at least to Mimic (v13.x.x) or even better to Nautilus (v14.x.x) now.
```

Comment:
I'm not sure this text change clarifies anything, and it seems to chop up the statement flow.